plugin/output/kafka/kafka.go

```diff
@@ -159,8 +175,9 @@ func (p *Plugin) out(workerData *pipeline.WorkerData, batch *pipeline.Batch) {
 		for _, e := range errs {
 			p.logger.Errorf("can't write batch: %s", e.Err.Error())
 		}
+		stats.GetCounter(subsystemName, sendErrorCounter).Add(float64(len(errs)))
```

Comment:
Why did you decide not to do a similar check here?
https://github.com/ozontech/file.d/pull/50/files#diff-ae9e68585530e0cefac2588a53be8e80e41ff31d9f37ae0cbf67bc1e971c45c5R237